src/toil_lib/tools/variant_manipulation.py

```diff
@@ -0,0 +1,335 @@
+import os
+import subprocess
+
+from toil_lib.programs import docker_call
+
+
+def gatk_select_variants(job, mode, vcf_id, config):
+    """
+    Isolates a particular variant type from a VCF file using GATK SelectVariants.
+
+    :param JobFunctionWrappingJob job: passed automatically by Toil
+    :param str mode: variant type (i.e. SNP or INDEL)
+    :param str vcf_id: VCF FileStoreID
+    :param Namespace config: Configuration and shared FileStoreIDs
+    :return: VCF FileStoreID
+    :rtype: str
+    """
+    job.fileStore.logToMaster('GATK SelectVariants: %s' % mode)
+
+    inputs = {'genome.fa': config.genome_fasta,
+              'genome.fa.fai': config.genome_fai,
+              'genome.dict': config.genome_dict,
+              'input.vcf': vcf_id}
```

Comment:
Same comment as above — If you're going to require a config, why have a separate argument just for this ID?